draft-ietf-rats-corim.md

```diff
@@ -2039,7 +2039,8 @@ After context initialization, additional inputs are held back until appraisal pr
 
 All available CoRIMs are collected.
 
-CoRIMs that are not within their validity period, or that cannot be associated with an authenticated and authorized source MUST be discarded.
+CoRIMs that are not within their validity period, have been revoked by an authorized source, or are
```

Comment:
CoRIM tags are discarded if:
* If they are expired
* that have been revoked
* are not associated with an authenticated source